is-valid/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use failure::Error;
+use std::io::stdin;
+
+use hpos_config_core::{config::Admin};
```

Comment:
You are using the wrong data structure for validation here. You want `hpos_config_core::Config`.

```suggestion
use hpos_config_core::*;
```